src/backend/main.py

```diff
@@ -15,32 +18,47 @@
 
 load_dotenv()
 
+ORIGINS = ["*"]
```

Comment:
let's constrain this before merging - or allow someone to constrain via configuration?

Reply:
I just capitalized the const, but we originally had `origins = ["*"]` set to streamline the deployments, without it a lot of the cloud deployments would require an extra step of modifying this in code, which we thought wasn't great UX